app/services/entitlement/feature_update_service.rb

```diff
@@ -70,6 +70,12 @@ def update_privileges
           create_privilege(code, privilege_params)
         else
           privilege.name = privilege_params[:name] if privilege_params.key?(:name)
+
+          if privilege_params.dig(:config, :select_options)
+            privilege.config["select_options"] ||= []
+            privilege.config["select_options"] |= privilege_params[:config][:select_options]
```

Comment:
This append behavior is confusing to me. Shouldn't we allow to provide a superset of the existing options ? For instance, if we have `[one two]`, you can provide `[one two three]`.

Reply:
This forces you to resend all values to add one. but it's a good point, maybe this should depend on the `partial` args!

Reply:
> This forces you to resend all values to add one

TBH I've never seen an endpoint when sending a value appends to existing ones. Usually, when that happens you have an `append` key or something similar.

But if you're OK with this behavior, then it's all good 👍